### ISSUE TITLE
Free orders return a 422 status code due to zero amount.

### ISSUE THREAD

**Issue by CosmicCoder96** [fossasia/open-event-server#5724]
Placing a free order results in a status of 422 because it fails the validation specified in the schema.

@iamareebjamal Is the validation intentionally for > 0 ? I am not aware, if it is, we can close this issue. Else it can be fixed if I change the comparison to >=0.

It's defined here:
https://github.com/fossasia/open-event-server/blob/8a8af7f9d8b08f86f2211dd1faf0cedbf455a17b/app/api/schema/orders.py#L50

To sum up: Should the amount for free orders be `null` or `0`.

**Comment by shreyanshdwivedi** [comment]
Working on this
